packages/udaru-core/.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
not sure about completely removing all control over installed package versions. More than that i would even add shrinkwrap to have precise control of the versions (at least on core and plugin). For example the pbac module does not respect semver and published a breaking change with minor version.
What do you think?

Reply:
I'm not sure tbh, I took direction on this from https://github.com/nearform/pathfinders/blob/master/nearform-opensource-guidelines.md and this in particular: https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514.
I think for pbac we should lock it down completely, and also any other modules that we know don't respect pbac. Very open to feedback on this.

Reply:
pbac is locked on fixed version, we'll have no issues here.
I'll ask around to see what are the practices on projects.